iati/home/tests/factories.py

```diff
@@ -0,0 +1,22 @@
+"""Factories for middleware tests."""
+import factory
+from factory.faker import faker
+from wagtail.documents.models import Document
+
+
+def random_filename():
+    """Use factory.faker to generate a random file name which includes an uppercase character."""
+    filegen = faker.Faker()
+    return filegen.random_uppercase_letter() + filegen.file_name()
```

Comment:
here you could use a string method instead of prepending a random letter to make mixed case - `return filegen.file_name().title()` or `return filegen.file_name().capitalize()`